static/app/views/explore/multiQueryMode/locationUtils.tsx

```diff
@@ -130,8 +131,9 @@ function getUpdatedLocationWithQueries(
   location: Location,
   queries: WritableExploreQueryParts[] | null | undefined
 ) {
+  let targetQueries: string[] | null = null;
```

Comment:
nit: I think you can simplify this even further using a ternary operator now that the else if condition is removed. Something like ```const targetQueries = defined(queries) ? queries.map(query => ...) : null;```